### PR TITLE
python-eventlet & python-greenlet: add new packages

### PR DESCRIPTION
**Pull request by ja-pa**
Maintainer: me
Compile tested: Turris Omnia (TOS6), OpenWrt master
Run tested: Turris Omnia (TOS6), OpenWrt master

Description:
This PR adds new package python-eventlet and its dependecy python-greenlet.

Eventlet is a concurrent networking library . [more info](https://eventlet.net/)

Greenlets are lightweight coroutines for in-process concurrent programming.  [more info](https://pypi.org/project/greenlet/)

### CODE REVIEW

File: lang/python/python-greenlet/Makefile [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-greenlet
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=Lightweight coroutines for in-process concurrent programming.
```
Comment (BKPepe) [inline lang/python/python-greenlet/Makefile:29]:
```suggestion
  TITLE:=Lightweight coroutines for in-process concurrent programming
```

**Comment by ja-pa** [comment]
Rebased
python-eventlet updated to 0.30.2

**Comment by BKPepe** [comment]
@jefferyto and @commodo can you I ask to review this, please?

**Comment by commodo** [comment]
i remember the greenlet package;
it was out-of-tree for a while, but required some tuning for the Python package to build it;

i think i broke it a while :p
https://github.com/openwrt/packages/issues/518

just as a note; it seems to compile on MIPS from what i can tell;
@kostko is this interesting for you these days?

**Review by commodo — APPROVED** [review]
from my side this looks good;
we can probably wait a while to see if we get any replies on the MIPS side;
but if there is some hurry to have this available sooner for testing, i don't mind to merge and bump the PKG_RELEASE for MIPS;

**Pull request merged**